charts/datadog-operator/templates/_helpers.tpl

```diff
@@ -87,4 +87,17 @@ Check operator image tag version.
 {{- else -}}
 {{ "1.8.0" }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Returns probe definition based on user settings and default HTTP port.
+Accepts a map with `port` (default port) and `settings` (probe settings).
+*/}}
+{{- define "probe.http" -}}
+{{ if or .settings.httpGet .settings.tcpSocket .settings.exec }}
```

Comment:
I haven't tested but I doubt Operator can respond to tcpSocket and exec probe requests, so not sure if we need these.